src/localize.ts

```diff
@@ -19,6 +21,7 @@ const languages: Record<string, unknown> = {
     "pt-BR": pt_BR,
     sv,
     "zh-Hans": zh_Hans,
+    "nl": nl,
```

Comment:
```suggestion
    nl,
```
No need to repeat the `nl` key.

Reply:
I edited this, it should look better now.